src/main/js/helpers/shapeSVG.js

```diff
@@ -15,3 +15,24 @@ export const getSVGObject = (shape, { posX, posY, scale }) => {
     d3Shape.options.scale = d3Shape.options.scale * scale;
     return d3Shape;
 };
+
+/**
+ * Returns an object to create a svg circle for event data point
+ *
+ * @private
+ * @param {object} shape - SHAPE object
+ * @param {number} eventShape - properties of event
+ * @param {number} scaleFactor - Scale factor for datapoint
```

Comment:
I don't see scaleFactor as parameter name here: A third parameter is just an object which as posX, posY and scale